tvm-disasm-cli/src/main/kotlin/org/ton/PrettyPrint.kt

```diff
@@ -0,0 +1,45 @@
+package org.ton
+
+import org.ton.bytecode.*
+import kotlin.reflect.full.memberProperties
+
+fun prettyPrint(disassembledFile: TvmContractCode) {
+    println("Main method instructions:")
+    printInstructions(disassembledFile.mainMethod.instList)
+
+    println("\nMethods instructions:")
+    disassembledFile.methods.forEach { (methodId, method) ->
+        println("\nMethod ID: $methodId")
+        printInstructions(method.instList)
+    }
+
+}
+
+fun printInstructions(instList: List<TvmInst>, indent: String = "") {
+    instList.forEach { inst ->
+        val type = inst.mnemonic
+        val operandsString = extractOperands(inst)
+
+        when (val firstOperand = inst::class.memberProperties
+            .firstOrNull { it.name !in IGNORED_PROPS }
+            ?.getter?.call(inst)
+        ) {
+            is TvmInstList -> {
+                println("$indent$type <{")
+                printInstructions(firstOperand.list, indent + "  ")
+                println("$indent}>")
+            }
```

Comment:
There are instructions with several continuations (for example, `IFREFELSEREF`. Look at interface `TvmContOperandInst`: all instructions that have continuations as their operands should be subclasses of it).